app/src/main/java/gov/hhs/cdc/trustedintermediary/domainconnector/DomainResponseHelper.java

```diff
@@ -0,0 +1,62 @@
+package gov.hhs.cdc.trustedintermediary.domainconnector;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import gov.hhs.cdc.trustedintermediary.wrappers.formatter.Formatter;
+import gov.hhs.cdc.trustedintermediary.wrappers.formatter.FormatterProcessingException;
+import java.util.Map;
+import java.util.Optional;
+import javax.inject.Inject;
+
+/** Provides helper methods for constructing a response to return to the client. */
+public class DomainResponseHelper {
+    static final String CONTENT_TYPE_LITERAL = "Content-Type";
```

Comment:
What are your thoughts on making this private?

Reply:
What's very interesting is that the unit tests seem to pass even if they were changed from package private to `private`.  That's not how "normal Java" works.  It shouldn't even work for `protected`.  Yet it does.  Interesting, I wonder what Groovy or Spock is doing to allow this.